packages/es-client/search.js

```diff
@@ -252,75 +252,6 @@ class BaseSearch {
     return response.detail !== recordNotFoundString;
   }
 
-  async granulesStats(key, value) {
```

Comment:
I didn't find any references to this method. It looks like this may have either been deprecated or partially replaced by the `Stats` class.